### DISCUSSION TITLE
Getting started with Discussions in the SamagraX community

### DISCUSSION THREAD

**Discussion by SaketSinha** [Samagra-Development/SamagraX-community#1] [General Discussions]
Welcome to Discussions on **SamagraX**. Please go through this post to understand how to engage via this forum.

We are a small group of highly ambitious team members who are trying to do it all, creating and maintaining open-source components, products, and this community. The feedback and encouragement that you provide us will help us get better with time.

Please put your post in the right category:
1. **General Discussions** - All the open questions that do not fit in any other section go here.
2.  **Installations and Configuration** - This category helps people seek clarifications for questions that arise at the time of installation of any component
3. **Request for new features** - This category is for the community to discuss potential new features for components and. products wired together by SamagraX.
4. **Troubleshooting and clarifications** - This category is for members who have already set up but are facing trouble in making changes to any product or component.

While on this forum, if you know the person who should pay attention to your post, please mention them (for e.g. to get Chakshu's attention, send a message with @ChakshuGautam). If you don't know who should pay attention or are seeking the right person, the moderators will step in to mention the appropriate person who needs to see the post.

Let's be nice to each other and build this community together with kindness and collaboration.

**Comment by SaketSinha** [comment]
Relevant people to tag: 
Engineering - @ChakshuGautam @bharatkashyap @umangbholasamagra 
Product - @SaketSinha @aayushkedia95 @sukhpreetsamagra

**Comment by aayushkedia95** [comment]
For all UCI related discussions, please tag @aayushkedia95.
